setup.cfg

```diff
@@ -31,6 +31,7 @@ all =
     pylint>=2.5.0
     rope>=0.10.5
     yapf
+    autobahn[twisted,accelerate]>=21.11.1
```

Comment:
I suggest to remove this line because Twisted is a big dependency and it's not really necessary for the server to work. If people want ẁebsocket functionality, they can use the `[websocket]` extra requirement directly.